pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "aioworkers-pg"
+version = "0.2.1-alpha.0"
```

Comment:
Can I use the version from the commit tag?

Reply:
Yeah, you can use 

pip install aioworkers-pg==0.2.1-alpha.1
pip install aioworkers-pg==0.2.1a1

Reply:
I published it locally. So now I try to configure publishing with Github Actions.